lib/onlyoffice_telegram_bugzilla_notifications/bug_filter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module OnlyofficeTelegramBugzillaNotifications
+  # Class to filter bugs by parameter
+  class BugFilter
+    # Initialize the Filters object
+    # @param bugzilla [OnlyofficeBugzillaHelper] instance of bugzilla api
+    # @param config [Hash] Configuration options for filtering
+    # @param bug_id [Integer] ID of the bug to be filter
+    def initialize(bugzilla, config, bug_id)
+      @bugzilla = bugzilla
+      @config = config
+      @bug_id = bug_id
+      @bug_info = @bugzilla.bug_data(@bug_id)
```

Comment:
I don't like that we send @bugzilla object to this abstaction, this feels like spaghetti code

Correct me if I wrong - we already request bug data right now in Message class (line 10)
And with this code we have a second place to request bug_data 

So this code will make twice calls to Bugzilla server - this is bad, especially after there is some DDOS protection - the service can simply be banned

Please redo this abstraction without additional calls to bugzill (probably will need to move getting data from Message too, but not sure)